python/src/aiconfig/ChatCompletion.py

```diff
@@ -0,0 +1,47 @@
+"""
+wrapper around openai.ChatCompletion.create that will serialize prompts and save them to config 
+
+usage:
+
+    Normal Import:
+        ` import openai`
+
+    Modified:
+        ```
+        import openai 
+        
+        from aiconfig.ChatCompletion import create_and_save_to_config
+        openai.ChatCompletion.create = create_and_save_to_config
+        ```
+"""
+
+from aiconfig.Config import AIConfigRuntime
+import openai
+import asyncio
+
+aiconfig = AIConfigRuntime.create("")
+
+original_create = openai.ChatCompletion.create
+
+
+def create_and_save_to_config(*args, **kwargs):
+    response = original_create(*args, **kwargs)
```

Comment:
use kwargs to include a few aiconfig-specific properties:
* aiconfig (pass in the AIConfigRuntime object to use to save the completion params and result)
* aiconfig_path (pass in the path to save a new AIConfigRuntime object to)
* Maybe also aiconfig_name and aiconfig_description

Reply:
This change would make the completion call more complicated since you now need to modify each API call

For instance:

Before: `openai.ChatCompletion.create(completion_params)`

After: `openai.ChatCompletion.create(completion_params, arg1, arg2, arg3, ...)`

Is this the intended change?

Reply:
Incorporating changes in a new diff as discussed offlinem cc: @saqadri